features/developer_portal/admin/account/credit_card_details/stripe.feature

```diff
@@ -0,0 +1,18 @@
+Feature: Billing and Credit card details for Stripe
+  In order to pay for the service
+  As a buyer
+  I want to enter my billing and credit card details and keep them up to date
+
+  Background:
+    Given a provider has charging configured and visible
+    And the provider has testing credentials for Stripe
+
+  @javascript
+  @stripe
+  Scenario: Set credit card details
```

Comment:
Coming back to this PR again. How critical do you see adding UI tests for the fields other than cardholder name (the one that this PR adds)?

I remembered the issue :) Basically, we configure the Chrome driver in a way that all Internet requests fail (no host resolved). However, the Stripe JS code is being loaded from https://js.stripe.com/v3

I tried tweaking the `--host-resolver-rules` option in the following way:

```
  options.add_argument("--host-resolver-rules=MAP js.stripe.com js.stripe.com.localhost:#{ENV['PORT']}, MAP * ~NOTFOUND , EXCLUDE *.localhost")
```

in an attempt to redirect the requests to localhost. But I can't find a way to make it http instead of https, now getting the following error:
```
Failed to load resource: net::ERR_SSL_PROTOCOL_ERROR
```

![Screenshot from 2022-10-24 13-16-48](https://user-images.githubusercontent.com/1270328/197514720-dfedf07d-20b5-43f5-8bd6-85b957f8e060.png)

I would like to ignore this for now, to avoid spending much time on this. Maybe we can come back to this later.

Or maybe you have another idea of how to test this?

In any case, I'm going to add some more JS tests.

Reply:
I think we need to enable a proxy server to intercept requests from the browser 😖. We will do eventually, no need to waste more time on this right now.